config/locales/nl.yml

```diff
@@ -220,7 +220,7 @@ nl:
   applications:
     invalid_url: De opgegeven URL is ongeldig
   auth:
-    agreement_html: Wanneer je op registeren klikt ga je akkoord met <a href="%{rules_path}">onze gebruikersvoorwaarden</a> en <a href="%{terms_path}">ons privacybeleid</a>.
```

Comment:
I'm checking `*.nl.yml` and `nl.json` now. I saw you translating the majority of parts. I do not think this is a problem, but I also want other people's opinions.